airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/unstructured_parser.py

```diff
@@ -166,6 +193,10 @@ def _get_filetype(self, file: IOBase, remote_file: RemoteFile) -> Optional[FileT
     def _supported_file_types(self) -> List[Any]:
         return [FileType.MD, FileType.PDF, FileType.DOCX, FileType.PPTX]
 
+    def _get_file_type_error_message(self, file_type: FileType) -> str:
+        supported_file_types = ", ".join([str(type) for type in self._supported_file_types()])
+        return f"File type {file_type} is not supported. Supported file types are {supported_file_types}"
```

Comment:
Nonblocking, a minor nit...

I noticed in the unit test output that the rendered text is not ideal for user-facing error message, due to referencing the internal class names instead of user-facing names. Based on the unit test output, the rendered output would be:

> ...Supported file types are FileType.MD, FileType.PDF, FileType.DOCX, FileType.PPTX

Ideally, we'd reference the file type or extension without the internal class name, but even as is, the error message is clear enough to be interpretable, and I wouldn't block on this. On digging deeper, I see these are defined in the Unstructured library, and I don't know what members would render a user-facing name like either "Markdown" or "*.md". 🤷